code/__HELPERS/atoms.dm

```diff
@@ -326,7 +326,7 @@ rough example of the "cone" made by the 3 dirs checked
  * * pass_args = pass_flags given to dummy object to allow it to ignore certain types of blockades.
  */
 /proc/los_check(atom/movable/user, mob/target, pass_args = PASSTABLE|PASSGLASS|PASSGRILLE, datum/callback/mid_check)
-	var/turf/user_turf = user.loc
+	var/turf/user_turf = get_turf(user)
```

Comment:
Doesn't this imply mobs inside closets and stuff will have LOS with targets? That seems incorrect

Reply:
its used for medbeams and the cosmic heretic beam that copies from medbeams its not an issue

Reply:
I feel like it should be using `get(src, /atom/movable)` for the medibeam or something

Reply:
i do not think theres any difference if you do it from the turf of the user or from the user
you can see out of closets
you can see out of mechas
actually you can see out of all objects

Reply:
ok fuck it plan B

Reply:
get(src, /atom/movable) not work because it would immediately return whatever you put as the first argument because mechs, items, mobs, are a movable

Reply:
I was thinking of `get_atom_on_turf` which gets the topmost movable

Reply:
should be done